portfolio.js

```diff
@@ -0,0 +1,56 @@
+const hamburger = document.querySelector('.bars');
+const overlay = document.querySelector('.open');
+const cancel = document.querySelector('.close');
+
+hamburger.addEventListener('click', () => {
+  overlay.style.display = 'block';
+});
+
+cancel.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
+
+const dynaaa = document.createElement('a');
+dynaaa.href = 'index.html#works';
+const dynaab = document.createElement('a');
+dynaab.href = 'index.html#about';
+const dynaac = document.createElement('a');
+dynaac.href = 'index.html#contact';
+const dynahr = document.createElement('hr');
+const dynahr1 = document.createElement('hr');
+const dynahr2 = document.createElement('hr');
+const dynadiv = document.createElement('footer');
+const dynahr3 = document.createElement('hr');
+dynadiv.className = 'dynadiv';
+dynaaa.className = 'dynaaa';
+dynaab.className = 'dynaab';
+dynaac.className = 'dynaab';
+dynahr.className = 'rule';
+dynahr1.className = 'rule';
+dynahr2.className = 'rule-8';
+dynahr3.className = 'rule-7';
+
+dynaaa.textContent = 'Portfolio';
+dynaab.textContent = 'About';
+dynaac.textContent = 'Contact';
+
+overlay.appendChild(dynaaa);
+overlay.appendChild(dynahr);
+overlay.appendChild(dynaab);
+overlay.appendChild(dynahr1);
+overlay.appendChild(dynaac);
+overlay.appendChild(dynahr2);
+overlay.appendChild(dynadiv);
+overlay.appendChild(dynahr3);
```

Comment:
- [OPTIONAL] Great job here 💯 However, I suggest using for loop instead of repeating the same code `DRY` and to Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). 👍

> For example 💡
```
const array = [dynaaa, dynahr, dynaab, dynahr1, dynaac, dynahr2, dynadiv, dynahr3];

array.forEach((item) => {
  overlay.appendChild(item);
});
```